tests/integration/test_bayesian_optimization.py

```diff
@@ -83,7 +83,7 @@
                 num_query_points=2,
             ),
         ),
-        (15, TrustRegion()),
+        (15, TrustRegion()),  # type: ignore
```

Comment:
Why this?

Reply:
this actually isn't needed in python 3.8. mypy is too eager to infer the type of this list as `tuple[int, AcquisitionRule[TensorType]]`, so when it gets to this line, it doesn't type check

Reply:
I could cast it but I though that'd be more messy than it's worth

Reply:
haha lol ok so mypy's decided to fail with unused `# type: ignore` if i have it but a type error if I don't. `cast` it is